conf/prod.routes

```diff
@@ -23,6 +23,7 @@ GET        /hmrc-healthcheck                          uk.gov.hmrc.helptosavestub
 
 # GetPAYEPErsonDetails
 GET        /pay-as-you-earn/02.00.00/individuals/:nino       uk.gov.hmrc.helptosavestub.controllers.PayePersonalDetailsController.getPayeDetails(nino)
+GET        /if/pay-as-you-earn/02.00.00/individuals/:nino       uk.gov.hmrc.helptosavestub.controllers.IFPayePersonalDetailsController.getPayeDetails(nino)
```

Comment:
Why is there only one new route, but two controllers?

Was there truly no way to reuse the existing DES controller?

Reply:
Well I think DES controller can be reused, it better to keep them separate though (avoid the 'ifs'). If you want DES controller to be re-used that can be done.